code/mob/living/carbon/human/monkeys.dm

```diff
@@ -256,6 +256,31 @@
 		theft_target.attack_hand(src)
 		src.a_intent = src.ai_default_intent
 
+	hear_talk(mob/M as mob, messages, heardname, lang_id)
+		if (!isdead(src) && !isunconscious(src) && messages)
+			if (M.singing)
+				if (M.singing == "bad" || M.singing == "loud")
+					if (rand(4) < 1)
+						// monkey is angered by singing
+						was_harmed(M)
+						spawn(5)
+							src.visible_message("<B>[name]</B> is angry at [M] for their [M.singing] singing!", 1)
+							src.say(pick("Must take revenge for insult to music!", "I now attack you like your singing attacked my ears!"))
+					else
+						spawn(5)
```

Comment:
unnecessary-seeming spawn

Reply:
don't want the monkey annoyed text to appear before the singing text